pkgs/development/tools/gojq/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, testVersion, gojq}:
+
+buildGoModule rec {
+  pname = "gojq";
+  version = "0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "itchyny";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hdrxd1eyr5y2G1LsQgmih7H6I3Dsbo0qmXr2WhpeghQ=";
+  };
+
+  vendorSha256 = "sha256-1wN1nMpvkkrKISuI9tCxX8nL73TQr9snHS9A1UWCArQ=";
+
+  CGO_ENABLED = 0;
```

Comment:
Is there a reason this needs to be packaged with `CGO_ENABLED = 0;`? If so please add a comment.

Reply:
Thanks for the clarification.